Patch brake parameter

crisvdn opened the pull request:
This is a patch so that the brake function will automatically convert a negative value to a postive value.
The brake function won't write negative values.

crisvdn committed:
Changed parameter of the brake functions. to uint8_t

The registers of the qik board called by the Brake functions doesn't accept negative values. uint8_t would prevent any issues with potentionally calling the function with a negative value.

crisvdn committed:
Merge branch 'release' into patch-brake_parameter

crisvdn committed:
Merge branch 'master' into patch-brake_parameter

crisvdn committed:
merge commit

crisvdn committed:
Merge branch 'release' into patch-brake_parameter

TGAssen reviewed (commented):
Fix doesn't convert a negative to a positive while keeping the same value. I wonder if this would cause issues with the range of parameters mentioned in the description of the functions.

crisvdn committed:
Changed brake functions to limit from 0 to 127, edited test cases to match functions

Edited the brake functions to limit the value from 0 to 127. If a negative value is passed, it will convert to a uint8_t and thus positive number. It will be limited between 0 and 127. Thats why the tests had to be adjusted to check with 127.

crisvdn committed:
Added doxygen for brake functions

TGAssen reviewed (approved):
Code fixes what is said it fixes. Looks good!
